### DISCUSSION TITLE
Token cost is counted even if langchains LLM cache is utilized

### DISCUSSION THREAD

**Discussion by arthurGrigo** [langfuse/langfuse#2111] [Support] [answered]
Not sure if this behaviour is intended as it is but when I use langchains LLM cache and get_openai_callback() it shows me total cost of 0$ for prompts I executed before. 

In langfuse I see the cost of the prompt as it would have been sent so the API.

@marcklingen Bug or feature?

**Comment by marcklingen** [comment, marked as answer]
I’d go for bug here, thanks for reporting!

Desiree behavior
- Token counts are not 0
- Cost is 0

Tracking this here: https://github.com/langfuse/langfuse/issues/2112
